cmd/clusterctl/README.md

```diff
@@ -71,6 +71,8 @@ $ kubectl --kubeconfig kubeconfig get machines
 $ kubectl --kubeconfig kubeconfig get machines -o yaml
 ```
 
+**NOTE:** There is no need to specify `--kubeconfig` if your `kubeconfig` was located in default directory under `$HOME/.kube/config` or if you already exposed env variable `KUBECONFIG`.
```

Comment:
Some minor tweaks:  
"was located in default directory" should be replaced with:
"was located in the default directory"

"you already exposed env variable" should be replaced with
"you have already exposed the env variable"

Reply:
+1 for the wording

Reply:
Done, thanks @bradtopol and @jichenjc